### PR TITLE
docs: glaredb@v0.4.0

### PR DESCRIPTION
**Pull request by greyscaled**

### CODE REVIEW

File: docs/releases/2023-august.md [hunk 1]
```diff
@@ -31,14 +31,25 @@ come in future releases.
...
 **Available in**: [GlareDB@v0.3.0], [GlareDB Cloud]
 
 A table can be created from a query:
 
 ```sql
-create table t1 as select * from generate_series(1, 5, 2);
+CREATE TABLE t1 AS SELECT * FROM generate_series(1, 5, 2);
+```
+
+**Available in**: [GlareDB@v0.4.0], [GlareDB Cloud]
```
Comment (greyscaled) [inline docs/releases/2023-august.md:44]:
```suggestion
**Available in**: [GlareDB@v0.4.0]
```

File: docs/releases/2023-august.md [hunk 2]
```diff
@@ -51,11 +62,16 @@ create table t1 as select * from generate_series(1, 5, 2);
   and gcs
 - All `*_scan` functions received support for passing multiple URLs
 
+**Available in**: [GlareDB@v0.4.0], [GlareDB Cloud]
```
Comment (greyscaled) [inline docs/releases/2023-august.md:65]:
```suggestion
**Available in**: [GlareDB@v0.4.0]
```

File: docs/releases/2023-august.md [hunk 3]
```diff
@@ -64,29 +80,39 @@ SELECT * EXCLUDE (id) FROM users;
...
 
 - [`glaredb local`] now has prettier output
 - [Python bindings] now include `show()` with pretty formatting and `close()`
   for gracefully closing connections.
 
-**Available in**: [GlareDB Cloud]:
+**Available in**: [GlareDB@v0.4.0]
+
+- Fixed `now()` returning the incorrect time
+- Fixed `show()` in Python not working in all environments
```
Comment (scsmithr) [inline docs/releases/2023-august.md:96]: Let's hope

**Review by scsmithr — APPROVED** [review]

File: docs/releases/2023-august.md [hunk 4]
```diff
@@ -31,14 +31,25 @@ come in future releases.
...
 A table can be created from a query:
 
 ```sql
-create table t1 as select * from generate_series(1, 5, 2);
+CREATE TABLE t1 AS SELECT * FROM generate_series(1, 5, 2);
+```
+
+**Available in**: [GlareDB@v0.4.0]
+
+Native tables now support UPDATE and [`DELETE`] statements:
```
Comment (greyscaled) [inline docs/releases/2023-august.md:46]: Link to UPDATE

**Pull request merged**